### PR TITLE
Allow getConstraints to return semantically equivalent constraint object

### PR DESCRIPTION
**Pull request by burnburn**
Fixes issue #360.

### CODE REVIEW

File: getusermedia.html [hunk 1]
```diff
@@ -4225,7 +4225,10 @@
               they were specified. Note that some of the optional
               ConstraintSets returned may not be currently satisfied. To check
               which ConstraintSets are currently in effect, the application
-              should use <code>getSettings</code>.</p>
+              should use <code>getSettings</code>.  The UA MAY return
+              normalized constraints with the identical effect in all
```
Comment (stefhak) [inline getusermedia.html]: is it clear what "normalized" means here? We only use that word at one other place in the document, and then it has to do with values of supported capabilities.

File: getusermedia.html [hunk 2]
```diff
@@ -4225,7 +4225,10 @@
               they were specified. Note that some of the optional
               ConstraintSets returned may not be currently satisfied. To check
               which ConstraintSets are currently in effect, the application
-              should use <code>getSettings</code>.</p>
+              should use <code>getSettings</code>.  The UA MAY return
```
Comment (alvestrand) [inline getusermedia.html]: Slightly outside the PR, but "To check which ConstraintSets are currently in effect" -> "To check which settings have been chosen based on the ConstraintSets". getSettings returns values for constraints that haven't been constrained, too.

File: getusermedia.html (hunk 1 above)
Comment (alvestrand) [inline getusermedia.html]: Suggest deleting the word. Perhaps "MAY return a constraint set that has the identical effect in all situations as the applied constraint, but does not need to return an identical constraint set."

File: getusermedia.html (hunk 2 above)
Comment (burnburn) [inline getusermedia.html]: Yes, this is outside the changes in the PR.  Better to make a new issue or PR for this.

**Comment by burnburn** [comment]
Please review again, @alvestrand and @stefhak .  I have made similar changes to what you suggested.

**Comment by stefhak** [comment]
This LGTM!

**Pull request merged**